### DISCUSSION TITLE
Possible to delete/create queues/topics?

### DISCUSSION THREAD

**Discussion by AndreAfonso87** [jmstoolbox/jmstoolbox#159] [Ideas]
Title says all. Is it possible to delete/create queues/topics?

**Comment by titou10titou10** [comment]
Short answer: no

This is not the philosophy of JMSToolBox, that uses QM administrative services to "discover" destinations

**Comment by AndreAfonso87** [comment]
Well in any case can it be added as enhancement for the future?
For development, sometimes its nice to be able to delete old queues without needing to go through script.

**Comment by titou10titou10** [comment]
JTB will never substitute to the administrative tools to manage destination in a queue manager
It's goal is to access destination by using the JMS protocol. JMS API does not include administrative functions (like JDBC for example that includes creating/deleting objects)
So if you want to create/delete queues or topics, you need to use the administrative tools coming with your queue nmanager
